### ISSUE TITLE
Shortcuts for changing open modes

### ISSUE THREAD

**Issue by AndrzejWawa** [ccbogel/QualCoder#663]
Hi, during work I noted that there is a need to change open modes frequently, I mean the "manage", "coding", and "reports. For example, when I use coding report there is sometimes a need to correct codings, or to note thoughts basing on reports or coding. Is it possible, to facilitate changing the modes through inserting convenient shortcuts, for example:
shift+z for "manage",
shift+x for "coding",
shift+c for reports.

It would be covenient, while all these buttons are close to each other, can be used by one hand, as well as correspond to the order of these tabs.

Two interesting facts: I found that it is very helpful to have opened many QC projects at the same time (now, I have open at least two projects all time), so, it is an advantage that there is no problem for this. Moreover, it is very convenient that there is no need for saving projects, as journals and codings are saved at continuous basis. Now, journals are the main places for me to make notes, more convenient than external text files.

**Comment by ccbogel** [comment]
OK to change tabs Ctrl+ tab
this is the normal behaviour. I have added an instruction in some of the tab widgets when the project is first opened.
![pic1](https://user-images.githubusercontent.com/5834638/175763611-9a3427d2-9c5f-46b4-9d9d-112b16cbfc20.png)

**Comment by ccbogel** [comment]
Its good to know the journals are very useful.

**Comment by AndrzejWawa** [comment]
Thanks! But is it possible to provide these dedicated shorcuts for earch of open mode (except action log)? It would be more convenient because it is often the sitation that you are changing between two particular modes, especially manage and reports or coding.

So, you are changing from right mode to left, and reverse, right to left and reverse. But the ctr-tab shorcut works only for one direction, as well as you have to move through all of tabs. So, to work with, e.g. manage and reports it is required go through:

reports=> action log => manage and manage => coding report, and again.

But with dedicated it is possible to go directly:

manage<=>reports

**Comment by ccbogel** [comment]
I have tried fixing this but I cannot make it work as you want. 
I could reduce the number of tabs - maybe combine coding and reports?

**Comment by AndrzejWawa** [comment]
No, I think that number is ok. It is good to have open coding window and report window at the same time. 

Could you add shift+Tab for moving tabs also in the reverse direction? I mean ctrl+Tab for the right, and shift+Tab for the left. If not, I would keep ctrl+Tab as you provided and close the issue.

**Comment by ccbogel** [comment]
I was checking the Qt key strokes. It is already there.
It is Ctrl Tab to go forward, 
Ctrl Shift Tab to  go backward

**Comment by AndrzejWawa** [comment]
Sorry, you right. So, you can only place additional info about shorcut for the backward.
